cloudhub-2/modules/ROOT/pages/_partials/deployment.adoc

```diff
@@ -96,6 +96,29 @@ To configure additional log levels and categories to include in logs:
 // end::configureLogging[]
 
 
+// tag::enableDistributedTracing[]
+
+== Enable Trace Data Collection
+
+Traces enable you to understand the execution path of a transaction in MuleSoft ecosystem by having full visibility into the execution path of API calls or application transactions. This feature exposes tracing information in OpenTelemetry standard from Mule runtime into all APM solutions.
+
+You can enable trace data collection for your CloudHub 2.0 applications running on Mule runtime version 4.6 or later. 
+
+To enable distributed tracing in your application deployments:
+
+. Click the *Monitoring* tab.
+. Select the *Enable tracing* checkbox.
+. Click *Deploy Application* if you are enabling the feature for a new deployment, or *Apply Changes* if you are editing an existing application.
+
+[NOTE]
+You can't enable trace data collection via Mule Maven plugin or Studio.
+
+Once you enable tracing, the Runtime Manager backend starts instrumenting traces for your application.
```

Comment:
Change to:
`Once you enable tracing, and redeploy your application, the Runtime Manager backend starts instrumenting traces for your application.`